wagtail/admin/views/pages/listing.py

```diff
@@ -321,6 +321,19 @@ def get_context_data(self, **kwargs):
         )
         return context
 
+    def get_ordering(self):
+        """
+        if page model has a get_default_order() method or default_order property
+        use it for the default ordering of the child pages
```

Comment:
```suggestion
        Use the parent Page's `get_admin_default_ordering` method.
```

Let's keep this short and sweet, no need to re-write what the method is doing, we could even avoid this docstring probably but happy to have it here.